lib/users/api.js

```diff
@@ -256,7 +244,7 @@ class API {
    * @throws {APIError} if the request was unsuccessful
    */
   async postDataAsURL(
-      url, filename, mimeType, size, encoding=undefined, ttl=undefined) {
```

Comment:
don't go crazy following the linter, which isn't quite configured to match the style of this repository. The style is to add an extra indent for multi-line things like this

Reply:
Can we fix the linter to match the style of this repository? It's going to drive me nuts because my code formats on-save based on the linter.

Reply:
Feel free to try to fix it! That'd be awesome

Reply:
FWIW the linter fix that was done here is in line with Google's style guide https://google.github.io/styleguide/jsguide.html#formatting-function-arguments

Reply:
Google JS style is two-space indents. It says 4 spaces in your link, but it really means "an additional two spaces beyond the normal indent". In this case, everything is already indented two spaces due to the class block, so it should be 6 spaces.

Yay linting.

Reply:
+1 for extra newline to distinguish this line from the `let uri` line below